capa/features/extractors/ts/signatures/cs.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
add doc/comment here what these entries represent, please

Reply:
could alternatively make this more explicit by adding a key, like `{ "imports": ... }`

Reply:
Resolved.